ui-tests/README.md

```diff
@@ -48,6 +51,12 @@ will be opened in your browser at the end of the tests execution; see
 [Playwright documentation](https://playwright.dev/docs/test-reporters#html-reporter)
 for configuring that behavior.
 
+### Noes for writing UI tests
+
+- If the UI tests fail, check the recording. If the video shows that no notebook is opened, it might  be that the tests are trying to open a notebook that doesn't exist, this might happen with notebooks in hidden folders
```

Comment:
might  be -> might be (double space)

Reply:
https://github.com/ploomber/jupysql-plugin/commit/6faf9f173c27a70ee75eb68c2caf58ab2b1aee2d